*** tests/A-017-006-M-ok.out	2014-12-03 23:20:53.000000000 +0000
--- tests/A-017-006-M-ok.outhyp	2014-12-09 08:52:29.298558984 +0000
***************
*** 45,51 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: Indique o nome da entrada: Identificador do utilizador: Menu de Shell
  1 - Listar
  2 - Listar entrada
  3 - Remover entrada
--- 45,52 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: Indique o nome da entrada: Identificador do utilizador: Mudar dono de entrada: Operação inválida: O utilizador 'root' não tem permissão para realizar a operação.
! Menu de Shell
  1 - Listar
  2 - Listar entrada
  3 - Remover entrada
***************
*** 61,67 ****
  Escolha uma opção: d - vader 32 .
  d - root 40 ..
  d - vader 16 private
! d w yoda 32 public
  Menu de Shell
  1 - Listar
  2 - Listar entrada
--- 62,68 ----
  Escolha uma opção: d - vader 32 .
  d - root 40 ..
  d - vader 16 private
! d w vader 32 public
  Menu de Shell
  1 - Listar
  2 - Listar entrada
***************
*** 88,94 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d w yoda 32 .
  d - vader 32 ..
  - - vader 23 filePriv
  - w vader 22 filePub
--- 89,95 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d w vader 32 .
  d - vader 32 ..
  - - vader 23 filePriv
  - w vader 22 filePub
