*** tests/A-017-003-M-ok.out	2014-12-03 23:20:53.000000000 +0000
--- tests/A-017-003-M-ok.outhyp	2014-12-09 08:52:28.950558624 +0000
***************
*** 70,76 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d w yoda 32 .
  d - vader 32 ..
  - - vader 23 filePriv
  - w vader 22 filePub
--- 70,76 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d w vader 32 .
  d - vader 32 ..
  - - vader 23 filePriv
  - w vader 22 filePub
